test/tests.py

```diff
@@ -648,7 +648,7 @@ class options(object):
                 self.cfilename + ".o",
                 module_id_base + ".o",
                 "-L"+join(host_path(), "bin", "py3")] + ldflags + \
-                ["-lsimics-common", "-lvtutils"]
+                ["-lsimics-common", "-lvtutils", "-lpthread"]
```

Comment:
I realized that `-lpthread` poses some problems: looks like the build system doesn't pass -lpthread by default (added to our local simics-base build env by `config/env.mk:35`). I don't understand why this doesn't cause t1000 to fail (perhaps the linker resolves pthread as a transitive dependency?).
Unsure what to do about this dependency; perhaps we could let -lvtutils or -lsimics-common re-export trampolines to pthread? or just add -lpthread as an explicit dep to all modules?

In any case, if we add a -lpthread dep, then it's a source of errors, so it deserves a releasenote entry even if we solve it in standard makefiles.